packages/vike-react-chakra/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Let's try to minimize this to the strict minimum while keeping `"sctrict": true`.

We'll copy-paste this a lot for future `vike-react-*` packages, so it's worth it to minimize it.

Reply:
Is it okay like this https://github.com/vikejs/vike-react/pull/151/commits/6f58d8e904a30768a4f1fa2f68d2366edd8b9fa7 ?